src/ansys/mechanical/env/run.py

```diff
@@ -53,20 +54,17 @@ def cli_find_mechanical(version: int):
 
     >>> find-mechanical -r 232
     """
-    # Get the version number
-    # TODO if modify line after fixing issue with atp for version
-    if not version:
-        exe = atp.get_mechanical_path()  # check for saved mechanical path
-        if exe:
-            version = atp.version_from_path("mechanical", exe)  # version is already int here
-        else:
-            exe, _version = atp.find_mechanical()
-            version = int(_version * 10)
-    else:
-        exe, _version = atp.find_mechanical(version=version)
-        version = int(_version * 10)
+    # checks for saved mechanical path else try to find installation path
+    exe = atp.get_mechanical_path(version)
+    if exe:
+        _version = atp.version_from_path("mechanical", exe)
+        if _version != version:
+            print(
+                f"Could not find cached Mechanical {version}. Using found {_version} instead.",
+                file=sys.stderr,
+            )
 
     aisol_path = os.path.dirname(exe)
-    print(version, aisol_path)
+    print(_version, aisol_path)
```

Comment:
should be `version` here, right?

Reply:
I found some other issues, I will push the changes later